client/rest-high-level/src/main/java/org/elasticsearch/client/Request.java

```diff
@@ -67,6 +69,8 @@
 import java.util.Objects;
 import java.util.StringJoiner;
 
+import static java.util.Collections.emptyMap;
```

Comment:
nit: can you revert the addition of the static import? It affects also other methods that you are not changing and that causes some noise in the PR.

Reply:
Will do 👍